src/resources/resources.controller.spec.ts

```diff
@@ -1,25 +1,271 @@
+                        lastName: expect.any(String),
+                        id: expect.any(String),
+                        avatar: expect.any(String),
+                    },
+                },
+                createdAt: expect.any(Date),
+                updatedAt: expect.any(Date),
+            });
+            expect(mockResourcesService.findAllResources).toHaveBeenCalledWith(
+                requestMock,
+                mockTeamId,
+            );
+        });
+        it("should throw notFound exception for invalid teamId", async () => {
+            mockResourcesService.findAllResources.mockRejectedValueOnce(
+                new NotFoundException(),
+            );
+            expect(
+                controller.findAllResources(requestMock, 9999),
+            ).rejects.toThrow(NotFoundException);
+            expect(mockResourcesService.findAllResources).toHaveBeenCalledWith(
+                requestMock,
+                9999,
+            );
+        });
+    });
+    describe("Update Resource", () => {
+        it("updateResource service should be defined", async () => {
+            expect(controller.updateResource).toBeDefined();
+        });
+
+        it("should update a resource", async () => {
+            mockResourcesService.updateResource.mockResolvedValueOnce(
+                mockUpdatedResource,
+            );
+
+            const result = await controller.updateResource(
+                requestMock,
+                1,
+                dtoUpdateMock,
+            );
+
+            expect(result).toEqual(mockUpdatedResource);
+            expect(mockResourcesService.updateResource).toHaveBeenCalledWith(
+                requestMock,
+                1,
+                dtoUpdateMock,
+            );
+        });
+        it("should throw notFound exception for invalid resourceId", async () => {
+            mockResourcesService.updateResource.mockRejectedValueOnce(
+                new NotFoundException(),
+            );
+            expect(
+                controller.updateResource(requestMock, 9999, dtoUpdateMock),
+            ).rejects.toThrow(NotFoundException);
+            expect(mockResourcesService.updateResource).toHaveBeenCalledWith(
+                requestMock,
+                9999,
+                dtoUpdateMock,
+            );
+        });
+    });
+
+    describe("Remove Resource", () => {
+        it("removeResource service should be defined", async () => {
+            expect(controller.removeResource).toBeDefined();
+        });
+
+        it("should delete a resource", async () => {
+            mockResourcesService.removeResource.mockResolvedValueOnce(
+                mockResource,
+            );
+
+            const result = await controller.removeResource(requestMock, 1);
+
+            expect(result).toEqual(mockResource);
+            expect(mockResourcesService.removeResource).toHaveBeenCalledWith(
+                requestMock,
+                1,
```

Comment:
try not to use magic numbers
```suggestion
                mockResource.id,
```